src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,66 @@
+import java.io.FileInputStream;
+import java.io.ObjectInputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.io.IOException;
+import java.io.EOFException;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.ObjectOutputStream;
+
+public class Storage {
+
+    private String filename;
+
+    /**
+     * Class constructor specifying the filename in the form of a String.
+     * @param filename the path of the file.
+     */
+    public Storage (String filename) {
+        this.filename = filename;
+    }
+
+    /**
+     * Loads the file if it exists. If not, create the file.
+     * @return ArrayList with the previous saved Tasks
+     * @throws IOException
+     * @throws ClassNotFoundException
+     */
+    public ArrayList<Task> load() throws IOException, ClassNotFoundException{
```

Comment:
This method returns something. Based on naming guidlines, should we change the method's name to a noun?